sources/LLETHER

```diff
@@ -1471,10 +1498,13 @@ Copyright (c) 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1990, 1991, 1992, 1993 b
                               HOSTNAMEP _ 'NILL])
 
 (\ETHEREVENTFN
-  [LAMBDA (DEV EVENT)                                    (* ; "Edited 15-Jan-88 01:30 by bvm")
-    (SELECTQ EVENT
-        ((NIL AFTERLOGOUT AFTERSYSOUT AFTERMAKESYS AFTERSAVEVM RESTART) 
-             (PROG (NDB TURNOFFNS TIMESET)
+  [LAMBDA (DEV EVENT)                                  (* ; "Edited  1-May-2021 19:23 by larry")
+    (PROG (NDB TURNOFFNS TIMESET)
+          (OR (SUBRCALL ETHER-AVAILABLE)
```

Comment:
I'm not convinced this is the right place to do this -- if you make a sysout on a machine with ethernet enabled, then you need to do the cleanup that follows (e.g., clearing `\PUP.READY` etc., clearing the ETHERPORTS) or you'll get the wrong behavior.

Reply:
good catch, updated. I made a separate issue for the printer problem.